btree/btree.c

```diff
@@ -2413,6 +2413,22 @@ static void bnode_op_fini(struct node_op *op)
  *  --------------------------------------------
  */
 
+/**
+ *  Structure of the directory entry.
+ */
+struct ff_dir_entry {
+	/**
+	 * uint16_t is sufficient to store the offset based on the assumtion
+	 * that the node size <= 64k.
+	 */
+	uint16_t ff_koff;
+	uint16_t ff_voff;
```

Comment:
```suggestion
	uint16_t ff_roff; /* Record offset */
```

You do not need 2 entries. You can use just one field that contain the offset which applies for both key and val. Since both are of same size the same offset will apply for both.

Reply:
Updated.